cmd/ris/risserver/server.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/bio-routing/bio-rd/routingtable"
 	"github.com/bio-routing/bio-rd/routingtable/filter"
 	"github.com/bio-routing/bio-rd/routingtable/locRIB"
+	"github.com/pkg/errors"
```

Comment:
I would wait for Go2 errors, or replace `github.com/pkg/error` with something Go2-error-proposal compatible.

Reply:
Do you know a package that is closer to Go2-error-proposal?

Reply:
https://godoc.org/golang.org/x/xerrors, which also claims to be partially merged into 1.13.